static/docs/get-started/agenda.md

```diff
@@ -24,7 +24,7 @@ Let the NLP nature of the example not to discourage you from using DVC in other
 Data Science areas. There was no strong reason behind picking the NLP area. On
 contrary, DVC is designed to be pretty agnostic of frameworks, languages, etc.
 If you have data files or data sets and/or you produce other data files, models,
-data sets and you want to:
+datasets, etc., then you may want to:
 
 - Capture and save those <abbr>data artifacts</abbr> the same way we capture
   code
```

Comment:
- [ ] The bullets starting with "Being able" below should use "Be able" instead.